commands/upgrade.js

```diff
@@ -25,8 +25,10 @@ ${cli.color.addon(db.name)} will be upgraded to a newer PostgreSQL version, stop
 
 This cannot be undone.`)
 
+  let body = flags.version ? {host: host(db), body: {version: flags.version}} : {host: host(db), body: {}}
```

Comment:
This is a little confusing since what you're calling `body` here is really more like "options", with a `body` field. Given that the version is the only thing different, maybe something like

```javascript
let body = {}
if (flags.version) {
  body.version = flags.version
}
yield cli.action(`Starting upgrade of ${cli.color.addon(db.name)}`, co(function * () {
  yield heroku.post(`/client/v11/databases/${db.id}/upgrade`, {host: host(db), body: body})
```
? Not a big deal, though.

Reply:
I can rename the variable, but we also do something similar to achieve an equivalent result in other commands which is how I modeled it https://github.com/heroku/heroku-pg/blob/master/commands/credentials/rotate.js#L45 I'm fine with doing either though as long as it's functionally equivalent